Desktop/python_exercise/operations.py

```diff
@@ -23,7 +23,7 @@
         if(num % 2 != 0):
             sum_of_odds += num
     print( sum_of_odds)
-
+    
```

Comment:
Remove this extra space as well.

Let's install the black formatter and apply on each of your question from now onwards.